lib/nfsc.js

```diff
@@ -125,14 +125,19 @@ class V3 {
      * authentication flavors are defined in Section 7.2 and
      * section 9 of [RFC1057].
      *
-     * @param {function} calllback(err: string, root_file_handle: Buffer);
+     * @param {function} calllback(err: null || {status: string},
+     *                             root_file_handle: Buffer);
      *                   On success, err is null. Continue execution with
      *                   the file handle of the root inode of the export.
      *                   On error, err contains information about the error.
      * @returns {undefined}
      */
     mount(callback) {
-        this.client.mount3(callback);
+        this.client.mount3((err, root_fh) => {
+            if (err)
+                return callback({status: err});
```

Comment:
Seems unusual to put the error in another object with the key `status`. I guess because you are checking for `res.status` in other places.